packages/manager/modules/pci/src/projects/project/notebooks/add/add.html

```diff
@@ -0,0 +1,145 @@
+<div>
+    <oui-header data-heading="{{:: 'pci_notebook_add_title' | translate }}">
+        <oui-guide-menu data-text="{{:: 'pci_project_guides_header' | translate }}">
+            <oui-guide-menu-item data-href="{{ $ctrl.guideUrl }}" data-external>
+                <span data-translate="pci_project_guides_header_all_guides"></span>
+            </oui-guide-menu-item>
+        </oui-guide-menu>
+    </oui-header>
+
+    <p data-translate="pci_notebook_add_description"></p>
+</div>
+
+<!--Banner messages-->
+<div class="mt-4" id="addMessages">
+    <oui-message data-type="info">
+        <span data-translate="pci_notebook_add_banner_object_storage_description"></span>
+        <a
+            class="btn btn-default btn-block"
+            data-ui-sref="pci.projects.project.storages.blocks.add"
```

Comment:
Use `ng-href`, with the link generated in the resolver instead. We stopped using the state names in the controller/html templates

Reply:
Done